### DISCUSSION TITLE
Are there any plans to make a web/desktop version?

### DISCUSSION THREAD

**Discussion by victor-marino** [enrique-lozano/Monekin#264] [Q&A] [answered]
I just found out about this app and it ticks almost all the boxes in terms of what I want in an expense tracker.

However, one thing I'm really missing is the ability to access the data from my PC as well.

I understand right now everything is local-only, so the first step would probably be to sync the data online somewhere.

Are there any plans to do this and eventually develop a web/desktop version?

P.S.: Any reason why the app isn't released for iOS as well? It should be quite straightforward, provided you have an Apple developer account of course.

**Comment by enrique-lozano** [comment, marked as answer]
Hi @victor-marino! Thanks for using the app.

There is already a version of the app for Windows that is updated at the same time as the Android one. You can find it in the [releases page](https://github.com/enrique-lozano/Monekin/releases). Just download the zip, unzip it wherever you want and double click on the app icon:

<img width="472" alt="image" src="https://github.com/user-attachments/assets/33d34af4-3ab1-4811-9c9f-5cccb59c8f15" />

I have pending to improve the download flow for Windows as well as add the link on the main page of the repo. 

Regarding the iOS app, I don't have an Apple developer account yet due to its high cost, but if the app gain popularity and the number of donations to the project is relatively high, I will consider making a financial effort in this regard :)

**Reply by victor-marino** [reply]
My apologies, didn't even see the Windows build! That's great.

I assume for now, the only way to sync between the Android and Windows clients would be to copy the .db export to our preferred cloud storage provider, then load that from the Windows app, right? (and vice-versa)

Honestly I don't think that's a bad system, as this allows people to use whatever sync method they prefer while retaining full control of their data and without the need to link the app to their Google account, etc.

I think perhaps the focus could be on automating the backups to cloud providers.

I'm a big fan of how [Portfolio Performance](https://www.portfolio-performance.info/en/) does it. In the PC client (native Java app), you just specify a local save path like any other desktop app, and changes are saved to that file. So if you pick a destination in your OneDrive/Google Drive/Dropbox folder, you're already backing things up to the cloud.

In the mobile app (Flutter), they've added support for a few providers, plus the option to pick files from the Android "Files" app, which also shows cloud folders:

![image](https://github.com/user-attachments/assets/06950aa8-8f29-42a7-a148-928370aeeddc)

I think there are Flutter packages for the main cloud providers, so maybe we can explore this if you agree with the approach.

I also have some experience with Flutter apps, so maybe I can try to look into this when I get some time :-)

**Reply by enrique-lozano** [reply]
Yes, your assumption is correct, if you have the app in multiple devices there is no way to sync between devices

The only possible way right now is importing the database file manually.
